app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CategoriesController < ApplicationController
+  load_resource
+  before_action :set_filter
+  def index
+    @books = Book.public_send(@filter).page(params[:page])
```

Comment:
move to scope to avoid dup with method show`.public_send(@filter).page(params[:page])`